Need to include unistd.h to define uid_t for musl libc.

--- snapper/Snapshot.h	2015-02-18 18:54:03.000000000 +0100
+++ snapper/Snapshot.h	2015-08-30 10:33:14.197150669 +0200
@@ -25,6 +25,7 @@
 
 
 #include <time.h>
+#include <unistd.h>
 #include <string>
 #include <list>
 #include <map>
